modules/graph-layers/src/widgets/view-control-widget.tsx

```diff
@@ -250,7 +250,6 @@ export class ViewControlWidget implements Widget<ViewControlProps> {
     // props.zoomInLabel = props.zoomInLabel || 'Zoom In';
     // props.zoomOutLabel = props.zoomOutLabel || 'Zoom Out';
     props.style = props.style || {};
-    this.props = props;
   }
 
   onAdd({deck}: {deck: Deck<any>}): HTMLDivElement {
```

Comment:
I think a good portion of this is handled by the abstract class now.

This function also adds a class "deck-widget-zoom" (which suggests it's a fork of ZoomWidget). That should be a good reference then for how this widget should be written in 9.2